ipn/ipnlocal/local.go

```diff
@@ -3535,6 +3536,77 @@ func (b *LocalBackend) setTCPPortsInterceptedFromNetmapAndPrefsLocked() {
 	b.setTCPPortsIntercepted(handlePorts)
 }
 
+// AddServeConfig receives a ServeConfig and adds TCP/Web mappings
+// to the local serveConfig, returns a read-only view or any errors
+//
+// b.mu must be held.
+func (b *LocalBackend) AddServeConfig(configIn ipn.ServeConfig) (*ipn.ServeConfigView, error) {
```

Comment:
oh! it is merging somehow. But how?

docs say "adds" but it also deletes in some cases. Document?